Vizzuality/heco-invest
# Frontend to use the SDG names returned by the endpoint

## Description

**In this PR:**  

- Updated `mockups/api/v1/enums` to include SDGs  
  _We're using the mockups to mock the api in Storybook stories_  
- Updated the following components to use the SDGs names returned by the `enums` endpoint:   
  - `containers/sdgs`  
  - `containers/forms/sdgs`  
  - `containers/forms/filters`  
  - `containers/project-details`  
- Updated the following pages to use the SDGs names returned by the `enums` endpoint:  
  - Investor public page  
  - Project public page   
- Removed the following no longer used mockup files:  
  - `enums.json`  
  - ~`sdgs.json`~ turns out it is needed

## Testing instructions

Verify that:  

- SDGs display correctly in:   
  - Investor public pages  
  - Project public pages  
  - Project details card in discover/projects  

- Choosing SDGs in forms works correctly in:  
  - New project form  
  - New investor form  

- SDGs filters display correctly in:  
  - Discover page search bar  

## Tracking

[LET-555](https://vizzuality.atlassian.net/browse/LET-555)
